db/migrations/postgres/000046_create_oauth_apps.up.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE IF NOT EXISTS oauthapps (
+    id VARCHAR(26) PRIMARY KEY,
+    creatorid VARCHAR(26),
+    createat bigint,
+    updateat bigint,
+    clientsecret VARCHAR(128),
+    name VARCHAR(64),
+    description VARCHAR(512),
+    callbackurls VARCHAR(1024),
+    homepage VARCHAR(256)
+);
```

Comment:
Another general question: I am wondering if it would be simpler for the table creation code to have the most recent version of the table. I am thinking it would be the same logic wise.

Reply:
But in that case we would lose the history right? And if we are going to include the down scripts with the reverse steps it wouldn't be consistent. cc @mgdelacroix